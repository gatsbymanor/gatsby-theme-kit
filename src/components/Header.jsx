import React from 'react';

export function Header() {

  const toggleModal = (event, name) => {
    event.preventDefault();
    // if (condition) {
    //
    // }

    // console.log('name', name);
  }

  return (
    <header id="header">
      <div class="logo">
        <span class="icon fa-diamond"></span>
      </div>
      <div class="content">
        <div class="inner">
          <h1>Dimension</h1>
          <p>A fully responsive site template designed by <a href="https://html5up.net">HTML5 UP</a> and released<br />
          for free under the <a href="https://html5up.net/license">Creative Commons</a> license.</p>
        </div>
      </div>
      <nav>
        <ul>
          <li><a onClick={ (event) => toggleModal(event, 'intro')}>Intro</a></li>
          <li><a onClick={ (event) => toggleModal(event, 'work')}>Work</a></li>
          <li><a onClick={ (event) => toggleModal(event, 'about')}>About</a></li>
          <li><a onClick={ (event) => toggleModal(event, 'contact')}>Contact</a></li>
        </ul>
      </nav>
    </header>
  );
}
