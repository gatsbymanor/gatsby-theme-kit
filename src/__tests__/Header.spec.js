import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

describe('<Header />', () => {
  const component = renderer.create(<Header />).toJSON();

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});