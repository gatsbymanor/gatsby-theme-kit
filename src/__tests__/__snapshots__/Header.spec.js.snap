// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `
<header
  className="alt"
  id="header"
>
  <span
    className="logo"
  >
    <img
      alt=""
      src={null}
    />
  </span>
  <h1>
    Stellar
  </h1>
  <p>
    Just another free, fully responsive site template
    <br />
    built by 
    <a
      href="https://twitter.com/ajlkn"
    >
      @ajlkn
    </a>
     for 
    <a
      href="https://html5up.net"
    >
      HTML5 UP
    </a>
    .
  </p>
</header>
`;
